### DISCUSSION TITLE
fastmri public leaderboard submission failed

### DISCUSSION THREAD

**Discussion by hellopipu** [facebookresearch/fastMRI#191] [Q&A] [answered]
Hi,

I'm not sure if it is appropriate to ask here.

I tried many times using dropbox url to submit my predictions, but it always gives me **ReadError: file could not be opened successfully**. I checked the zip files on my own machine, and I didn't see any problem. Could you help me with this?

**Comment by mmuckley** [comment]
Hello @hellopipu. You can ask about this question here. Could you create a new submission so that I can try to debug it?

**Reply by hellopipu** [reply]
yes, I submit it again.

**Reply by mmuckley** [reply]
Okay working on it now.

**Reply by mmuckley** [reply]
I don't want to post it here, but the link included with the submission generates a 404 Error.

**Reply by hellopipu** [reply]
hi, sorry to take your time. The link in last submission is somehow invalid; I will appreciate if  you can check my recent submission again.

**Reply by mmuckley** [reply]
Okay it's looking like there is an error with how the server backs up data to a database.

I'm working on resolving it - if you are all right with it please leave your submission up until I can resolve.

**Comment by mmuckley** [comment, marked as answer]
Hello @hellopipu, your submission should work now. We have some changes we need to make to the site due to how credentials are being used in our backends, but we'll try to wait until after ISMRM to do this.

**Reply by hellopipu** [reply]
Thank you, it works now.
